app/javascript/packs/admin/users/memberIndex.js

```diff
@@ -113,10 +113,10 @@ document.querySelectorAll('.js-export-csv-modal-trigger').forEach((item) => {
   item.addEventListener('click', () => {
     showWindowModal({
       title: 'Download Member Data',
-      contentSelector: '#export-csv-modal',
+      contentSelector: item.dataset.modalContentSelector,
```

Comment:
while I was reviewing our modal triggers I thought it made sense to make this small tweak